Automattic/jetpack
# WordAds: Add Global Privacy Control (GPC) support for CCPA

#### Changes proposed in this Pull Request:

The California Consumer Privacy Act (CCPA) requires that we recognize users' right to opt-out to sharing of their personal information by interpreting signals sent through browser controls. The [Global Privacy Control](https://globalprivacycontrol.org/) initiative is an attempt to standardize this browser signal. This PR adds support for the Global Privacy Control to WordAds.

#### Does this pull request change what data or activity we track or use?

No

#### Testing instructions:
* Create a Jetpack site with Ads enabled
* Enable the CCPA feature of Ads
* Visit the site from a browser with a California IP address (VPN if needed) or ...
* Set a cookie manually. Name: `usprivacy` Value: `1YNN` (signaling no opt-out)
* Run `__uspapi('getUSPData', 1, (data) => console.log(data));` from the JS console
* Verify the `uspString` value is `1YNN` (signaling no opt-out)
* Visit the same site with the Brave browser (GPC auto-enabled)
* Run `__uspapi('getUSPData', 1, (data) => console.log(data));` from the JS console
* Verify the `uspString` value is `1YYN` (signaling opt-out)
